test/Functional/CreateChargeTest.php

```diff
@@ -16,6 +16,15 @@ public function testChargeCard()
         ));
     }
 
+    public function testChargeNonChargeableCardFails()
```

Comment:
This test should have a docblock comment similar to the one I [described in the previous review](https://github.com/Provider/Stripe/pull/2#discussion_r117485617).